### PR TITLE
Fixes #23322: Add back RUDDER_VERIFY_CERTIFICATES to rudder.json

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/23322

7.X agents still read this value, we need to provide it. Usage in agent is removed in https://github.com/Normation/rudder-agent/pull/387 and https://github.com/Normation/rudder-plugins/pull/595.

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala [hunk 1]
```diff
@@ -146,7 +146,9 @@ class PrepareTemplateVariablesImpl(
         .toVariable(if (agentNodeConfig.agentType == AgentType.CfeCommunity) Seq("true") else Seq()),
       systemVariableSpecService.get("AGENT_TYPE").toVariable(Seq(agentNodeConfig.agentType.toString)),
       systemVariableSpecService.get("RUDDER_NODE_CONFIG_ID").toVariable(Seq(nodeConfigVersion.value)),
-      systemVariableSpecService.get("RUDDER_COMPLIANCE_MODE").toVariable(Seq(agentPolicyMode.name))
+      systemVariableSpecService.get("RUDDER_COMPLIANCE_MODE").toVariable(Seq(agentPolicyMode.name)),
+      // For compatibility with pre-8.0 agents
+      systemVariableSpecService.get("RUDDER_VERIFY_CERTIFICATES").toVariable(Seq("false"))
```
Comment (fanf) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala:151]: You need to revert what was in `SystemVariableSpecImpl` (https://github.com/Normation/rudder/pull/4982/files#diff-b2bb87fe54627ab29d48267c7a88d069948c01442c48aeeaeaa1104cb1e441a6) to have that working

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala (hunk 1 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala:151]: Perhaps you can create the spec it here in an ad-hoc way with a comment "for compat, will be removed in 8.1" (or when it's time)

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala (hunk 1 above)
Comment (amousset) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala:151]: The tests seem to pass without it, is it expected?

**Comment by amousset** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala (hunk 1 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala:151]: No ? (I'm not even sure why you didn't get a "missing system variable spec" error)

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**